app/src/main/java/org/jellyfin/androidtv/TvApp.java

```diff
@@ -132,8 +132,8 @@ public void onResponse(DisplayPreferences response) {
                     if (response.getSortBy() == null) response.setSortBy("SortName");
                     if (response.getCustomPrefs() == null)
                         response.setCustomPrefs(new HashMap<String, String>());
-                    if (app.equals(TvApp.DISPLAY_PREFS_APP_NAME))
```

Comment:
Are you sure `app` always equals `TvApp.DISPLAY_PREFS_APP_NAME` here?

Reply:
The issue is that it doesn't. The settings we want are part of the `emby` app(jellyfin-web). That line only allowed settings to be cached if it was part of `ATV` so I removed it to cache the `emby` settings so we don't have to fetch from server everytime.

Reply:
So this cache is removed entirely in #1011 but until that's merged we need a reasonable solution.
If we use multiple keys and want to be 100% sure it won't cause conflict we should add the app name to the cache too like you suggest, but that requires changing the getCachedDisplayPrefs function parameters for the app name.